docs/modules/transform-sync/how-it-works.md

```diff
@@ -0,0 +1,63 @@
+# How the Transform Synchronization Feature Module works
+
+## What is transform
+
+The transform of an entity describes its location and rotation. Worker-instances generally use it to run physics simulations or render entities for a client. Note that the SpatialOS representation of transform is _not_ the same as Unity's representation of transform.
+
+In the Transform Synchronization Feature Module, we represent the Transform as:
+
+```
+package improbable.transform;
```

Comment:
These are implementation details that might change at any time and users aren't supposed to know.

Reply:
In fact this is only what it was in the public branch at release. My internal branch had a different representation that was intended to be released later.

Reply:
Its not like its actually hidden in any way

Reply:
It does mean that this will likely go out of sync. Also eventually it will be incomprehensible to someone who doesn't know how it works already.

Reply:
It does increase the likelihood of that (until we can include or similar)